pyteal/compiler/subroutines.py

```diff
@@ -62,6 +63,32 @@ def findRecursionPoints(
     return reentryPoints
 
 
+def find_recursive_path(
+    subroutine_graph: Dict[SubroutineDefinition, Set[SubroutineDefinition]],
+    subroutine: SubroutineDefinition,
+) -> List[SubroutineDefinition]:
+    visited = set()
```

Comment:
the API of this function was sufficiently different from that of `depthFirstSearch()` that it was cleaner to just create a brand new function

Reply:
👍